_episodes/41-code-review.md

```diff
@@ -109,49 +83,6 @@ Since the cost of bug fixes grows in orders of magnitude throughout the software
 it is far more efficient to find and fix defects
 as close as possible to the point where they were introduced.
 
-There are several **code review techniques** with various degree of formality
-and the use of a technical infrastructure, including:
-
-- **Over-the-shoulder code review**
-  is the most common and informal of code review techniques and involves
-  one or more team members standing over the code author's shoulder
-  while the author walks the reviewers through a set of code changes.
-- **Email pass-around code review**
-  is another form of lightweight code review where the code author
-  packages up a set of changes and files and sends them over to reviewers via email.
-  Reviewers examine the files and differences against the code base,
-  ask questions and discuss with the author and other developers,
-  and suggest changes over email.
-  The difficult part of this process is the manual collection the files under review
-  and noting differences.
-- **Pair programming**
-  is a code development process that incorporates continuous code review -
-  two developers sit together at a computer,
-  but only one of them actively codes whereas the other provides real-time feedback.
-  It is a great way to inspect new code and train developers,
-  especially if an experienced team member walks a younger developer through the new code,
-  providing explanations and suggestions through a conversation.
-  It is conducted in-person and synchronously but it can be time-consuming
-  as the reviewer cannot do any other work during the pair programming period.
-- **Fagan code inspection**
-  is a formal and heavyweight process of finding defects in specifications or designs
-  during various phases of the software development process.
-  There are several roles taken by different team members in a Fagan inspection
-  and each inspection is a formal 7-step process with a predefined entry and exit criteria.
-  See [Fagan inspection](https://en.wikipedia.org/wiki/Fagan_inspection)
-  for full details on this method.
-- **Tool-assisted code review**
-  process uses a specialised tool to facilitate the process of code review,
-  which typically helps with the following tasks:
-  (1) collecting and displaying the updated files and highlighting what has changed,
-  (2) facilitating a conversation between team members (reviewers and developers), and
-  (3) allowing code administrators and product managers
-  a certain control and overview of the code development workflow.
-  Modern tools may provide a handful of other functionalities too, such as metrics
-  (e.g. inspection rate, defect rate, defect density).
-
-Each of the above techniques have their pros and cons and varying degrees practicality -
-it is up to the team to decide which ones are most suitable for the project and when to use them.
```

Comment:
I don't see how this section contributes to the goals, so removing this gives more time to focus on the kind of review system we're recommending - tool assisted review.

Reply:
I will make this adjustment as I think you're right, it is nice to have some other terms to look up, but with less detail.